app/pages/collections/list.cjsx

```diff
@@ -50,7 +50,7 @@ List = React.createClass
   listCollections: (props) ->
     query = {}
     if props.params?.collection_owner?
-      query.owner = props.params.collection_owner
+      query.current_user_roles = "owner,contributor,collaborator"
```

Comment:
This was when we thought that the difference between `params.collection_owner` and `params.profile_owner` was the route, with the former being our own page and the latter being someone else's. We have since discovered that is not the case. Using owner will def not work because we explicitly want to see collections that we do not own. 

I need a way to, on that page, determine whether I should use `current_user_roles` and view my own owned, collaborated, and contributed collections, OR use `owner` because I'm looking at someone else's and should only see their public, owned collections.